### DISCUSSION TITLE
Great this repo is not dead. :)

### DISCUSSION THREAD

**Discussion by MR-444** [jodendaal/OpenAI.Net#86] [General]
Hello, 
Indeed I am still using your Nuget Package, and I am glad that you continue working on it. :)

**Comment by jodendaal** [comment]
Thanks will be doing some more updates in a few weeks to handle functions and possibly a few other features that have since been added to the API. Also looking for contributors if you or anyone else is interested.

**Reply by MR-444** [reply]
Function calls would be awesome, as well as the assistance part (but I am hesitant to implement it myself because it's still in beta, and it doesn't seem very user-friendly yet.)
Contributing: I know the best software is the one you write yourself, but there are some alternatives on GitHub, like https://github.com/betalgo/openai. Maybe you should direct your efforts to this repo?
I chose your package a couple of months ago because my code was looking nearly identical to yours, so I stopped coding and opted for your package instead.
I think I can't contribute much, perhaps if I find some weird stuff, but your code seems very neat with high standards.
